File: test-automation-core/src/main/java/io/cucumber/testng/FeatureWrapperImpl.java
package io.cucumber.testng;

import io.cucumber.core.gherkin.Feature;
import lombok.Getter;

/**
 * Direct copy from the lib. Changing class visibility.
 */
@Getter
public class FeatureWrapperImpl implements FeatureWrapper {
	private final Feature feature;

	FeatureWrapperImpl(Feature feature) {
		this.feature = feature;
	}

	public String toString() {
		return "\"" + this.feature.getName() + "\"";
	}
}

File: test-automation-core-old/src/main/kotlin/io/cucumber/testng/FeatureWrapperImpl.kt
package io.cucumber.testng

import io.cucumber.core.gherkin.Feature
import io.cucumber.testng.FeatureWrapper
import lombok.Getter

/**
 * Direct copy from the lib. Changing class visibility.
 */
@Getter
class FeatureWrapperImpl internal constructor(val feature: Feature) : FeatureWrapper {
    override fun toString(): String {
        return "\"" + feature.name + "\""
    }
}